### PR TITLE
refactor: section

### PR DESCRIPTION
**Pull request by rohitp-bsf**
### Description
- refactor: section

### Screenshots
- NA

### Types of changes
- Converted Class-based components to functional-based.
- Refactored the edit JS file into render and settings.

### How has this been tested?
- NA

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/section/settings.js [hunk 1]
```diff
@@ -0,0 +1,1747 @@
...
+	RangeControl,
+	Button,
+	ButtonGroup,
+	BaseControl,
+	ToggleControl,
+} from '@wordpress/components';
+
+const ColumnResponsive = lazy( () =>
+	import(
+		/* webpackChunkName: "chunks/section/column-responsive" */ '../../components/typography/column-responsive'
```
Comment (sandeshjangam) [inline src/blocks/section/settings.js:24]: Use @Components alias here

File: src/blocks/section/settings.js [hunk 2]
```diff
@@ -0,0 +1,1747 @@
...
+	)
+);
+
+const GradientSettings = lazy( () =>
+	import(
+		/* webpackChunkName: "chunks/section/gradient-settings" */ '../../components/gradient-settings'
```
Comment (sandeshjangam) [inline src/blocks/section/settings.js:30]: Use @Components alias here

File: src/blocks/section/settings.js [hunk 3]
```diff
@@ -0,0 +1,1747 @@
...
+	)
+);
+
+const BoxShadowControl = lazy( () =>
+	import(
+		/* webpackChunkName: "chunks/section/box-shadow-control" */ '../../components/box-shadow'
```
Comment (sandeshjangam) [inline src/blocks/section/settings.js:36]: Use @Components alias here

**Review by sandeshjangam — CHANGES_REQUESTED** [review]
Need changes

File: src/blocks/section/settings.js (hunk 3 above)
Comment (rohitp-bsf) [reply src/blocks/section/settings.js]: Done!

File: src/blocks/section/settings.js (hunk 2 above)
Comment (rohitp-bsf) [reply src/blocks/section/settings.js]: Done!

File: src/blocks/section/settings.js (hunk 1 above)
Comment (rohitp-bsf) [reply src/blocks/section/settings.js]: Done!

File: src/blocks/section/inline-styles.js [hunk 4]
```diff
@@ -21,82 +20,91 @@ function inlineStyles( props, isEditor ) {
...
+		style[ 'border-style' ] = borderStyle;
+		style[ 'border-width' ] = generateCSSUnit( borderWidth, 'px' );
+		style[ 'border-color' ] = borderColor;
 	}
 
-	var position = backgroundPosition.replace( "-", " " )
+	const position = backgroundPosition.replace( '-', ' ' );
 
-	var section_width = "100%"
+	let section_width = '100%';
```
Comment (RajkiranGajananBagal) [inline src/blocks/section/inline-styles.js:87]: camelCase variables.

File: src/blocks/section/save.js [hunk 5]
```diff
@@ -2,63 +2,58 @@
...
 		contentWidth,
-		align
-	} = props.attributes
-
-	let block_controls_class = ""
+		align,
+	} = attributes;
 
-	if ( "full_width" == contentWidth ) {
+	let block_controls_class = '';
```
Comment (RajkiranGajananBagal) [inline src/blocks/section/save.js:21]: camelCase variables

File: src/blocks/section/style.scss [hunk 6]
```diff
@@ -9,52 +9,52 @@
...
 	.uagb-section__inner-wrap {
-		margin-left: auto;
-		margin-right: auto;
 		position: relative;
 		z-index: 2;
+		margin-right: auto;
+		margin-left: auto;
 	}
 
 	.uagb-section__overlay {
```
Comment (RajkiranGajananBagal) [inline src/blocks/section/style.scss:18]: We can combine common css in 
.uagb-section__overlay & .uagb-section__video-wra CSS

File: src/blocks/section/styling.js [hunk 7]
```diff
@@ -62,141 +60,240 @@ function styling( props ) {
 		boxShadowBlur,
 		boxShadowSpread,
 		boxShadowPosition,
-		gradientValue
-	} = props.attributes
+		gradientValue,
+	} = props.attributes;
 
-	var inner_width = "100%"
+	let inner_width = '100%';
```
Comment (RajkiranGajananBagal) [inline src/blocks/section/styling.js:66]: camelCase variables.

**Review by RajkiranGajananBagal — CHANGES_REQUESTED** [review]

File: src/blocks/section/inline-styles.js (hunk 4 above)
Comment (rohitp-bsf) [reply src/blocks/section/inline-styles.js]: done!

File: src/blocks/section/save.js (hunk 5 above)
Comment (rohitp-bsf) [reply src/blocks/section/save.js]: done!

File: src/blocks/section/style.scss (hunk 6 above)
Comment (rohitp-bsf) [reply src/blocks/section/style.scss:18]: done!

File: src/blocks/section/styling.js (hunk 7 above)
Comment (rohitp-bsf) [reply src/blocks/section/styling.js]: done!

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/section/save.js [hunk 8]
```diff
@@ -2,63 +2,58 @@
...
 		<CustomTag
 			className={ classnames(
 				className,
-				"uagb-section__wrap",
-				`uagb-section__background-${backgroundType}`,
-				block_controls_class,
-				`uagb-block-${block_id}`
+				'uagb-section__wrap',
+				`uagb-section__background-${ backgroundType }`,
+				blockControlsClass,
```
Comment (sandeshjangam) [inline src/blocks/section/save.js:37]: Attribute change will create a problem for old users.

**Review by sandeshjangam — APPROVED** [review]
@rohitp-bsf  Need

**Pull request merged**